scala/scala.bzl

```diff
@@ -644,6 +679,8 @@ scala_library = rule(
   attrs={
       "main_class": attr.string(),
       "exports": attr.label_list(allow_files=False),
+      "enable_dependency_analyzer": attr.bool(default=True, mandatory=False),
+      "dependency_analyzer_plugin": attr.label(default=Label("//plugin/src/main:dependency_analyzer"), allow_files=_jar_filetype, mandatory=False),
```

Comment:
I think you may need a full `@io_bazelbuild_rules_scala//plugin/...` name here since it may not be expanded correctly in remote repos.